apps/api/src/app/shared/services/cache/index.ts

```diff
@@ -1,3 +1,2 @@
 export * from './invalidate-cache.service';
 export * from './cache.service';
-export { CacheKeyPrefixEnum } from './keys';
```

Comment:
I would export the key builders so the imports would be simplified in the rest of the codebase.